Formula/demumble.rb

```diff
@@ -0,0 +1,22 @@
+class Demumble < Formula
+  desc "More powerful symbol demangler (a la c++filt)"
+  homepage "https://github.com/nico/demumble"
+  url "https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz"
+  sha256 "13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f"
+  license "Apache-2.0"
+  head "https://github.com/nico/demumble.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    bin.install "build/demumble"
```

Comment:
```suggestion
    system "cmake", "--install", "build"
```
Does this not work here?

Reply:
No, upstream has no CPack/install rules in the cmake file